### ISSUE TITLE
Create status commandlet

### ISSUE THREAD

**Issue by hohwille** [devonfw/IDEasy#758]
As a IDEasy user, I want to have a simple command so that I can quickly see outstanding notifications and warnings.

Currently we log such notifications and warnings on the fly while processing arbitrary commandlets.
However, there are also some commandlets that are considered to produce "processable output" that need to suppress such output.

Question: What commandelt should the end-user invoke in order to see if there are updates to `settings` or there are warnings for his configuration?
Answer: There is currently no clean answer this this question. You may call `ide help` and get the output on top of the help but need to scroll up in order to find it. You may call `ide -v` or `ide` or `ide get-version java` and notice that all of these have processable output and suppress the information. You might get the output if you add `--debug` option but then you also get lots of debug output. So you might call `ide install java` or something but just to get such information you do not want to invoke something like an installation.

So in other words: 
- [ ] We need a `StatusCommandlet` that can be called via `ide status` to get only such status information without applying any changes.
- [ ] I also propose that just typing `ide` should invoke `ideasy status` if we are inside an IDEasy project.
- [ ] Whenever I call this `StatusCommandlet` it will print me all warnings about my `settings` (e.g. duplicated or legacy variables) as well as potential information about updates to `settings` I should apply with `ide update`.
